### PR TITLE
Add function selectworkspace

### PR DESCRIPTION
**Pull request by lb-cuonghx**

### CODE REVIEW

**Comment by doantuann** [comment]
LGTM 👌

File: lib/src/hexabase.dart [hunk 1]
```diff
@@ -81,6 +83,20 @@ class Hexabase {
     return token != null;
   }
 
+  Future<HexabaseWorkspace> selectWorkspace(String workspaceId) async {
```
Comment (kondokentaro) [inline lib/src/hexabase.dart]: Please change method name "selectWorkspace" to "current".
I'd like to use the same name as the JS SDK.

**Review by kondokentaro — COMMENTED** [review]
@doantuann @lb-cuonghx 
Thank you very much!

File: lib/src/hexabase.dart (hunk 1 above)
Comment (lb-cuonghx) [reply lib/src/hexabase.dart]: @kondokentaro 
I have updated it, please check it for me ! 
Thank you!

Comment (kondokentaro) [reply lib/src/hexabase.dart]: @lb-cuonghx 
LGTM!
Thank you very much!

**Pull request merged**